pandas/tests/frame/test_reductions.py

```diff
@@ -1043,6 +1043,25 @@ def test_sum_bools(self):
         bools = isna(df)
         assert bools.sum(axis=1)[0] == 10
 
+    def test_sum_string_dtype_coercion(self):
+        # GH#22642
+        # Check that summing numeric strings results in concatenation
+        # and not conversion to dtype int64 or float64
+        df = DataFrame({"a": ["483", "3"], "b": ["94", "759"]})
+        result = df.sum(axis=1)
+        expected = Series(["48394", "3759"])
```

Comment:
Could you use `pytest.mark.parametrize` like  

```python
@pytest.mark.parametrize("input_data,expected_data", [[{"a": ...}, ["48394" ...]] ,...])
```

so we don't have to repeat 3 separate setups?